config.json

```diff
@@ -1801,26 +1801,26 @@
       {
         "groupId": "com.squareup.okhttp3",
         "artifactId": "logging-interceptor",
-        "version": "4.6.0",
-        "nugetVersion": "4.6.0",
+        "version": "4.5.0",
```

Comment:
Version downgrade

Reply:
I specifically asked to review and merge this PR as 1st (or after last GPS one)

From discord:

> is there any way to commit #730 and #731 first?  the issue is that #729 also contains the changes in the other 2